modules/org-policies/iam.tf

```diff
@@ -33,3 +33,13 @@ module "service_account_key_policy" {
   policy_type = "boolean"
   enforce     = true
 }
+
```

Comment:
might want to update testcase by adding another constraint to check for in the "simple" example:

https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse/blob/main/test/integration/simple-example/controls/gcloud.rb#L21